### PR TITLE
Add support for building centos-8 and Amazon Linux 2 packages

### PR DESCRIPTION
**Pull request by neilcook**

### CODE REVIEW

File: builder-support/dockerfiles/Dockerfile.target.centos-8 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+RUN yum install -y epel-release
+RUN dnf -y install dnf-plugins-core
+RUN dnf config-manager --set-enabled powertools
+
+# Do the actual rpm build
+@INCLUDE Dockerfile.rpmbuild
+
+# Do a test install and verify
+# Can be skipped with skiptests=1 in the environment
+#@EXEC [ "$skiptests" = "" ] && include Dockerfile.rpmtest
```
Comment (Habbie) [inline builder-support/dockerfiles/Dockerfile.target.centos-8:16]: Is this commented on purpose?

File: .travis.yml [hunk 2]
```diff
@@ -78,7 +92,6 @@ jobs:
       - cd ..
       - bash docker_push.sh
       - cd ..
-      - ./trigger_policy_build.sh
```
Comment (Habbie) [inline .travis.yml:81]: Is this removed on purpose?

File: builder-support/dockerfiles/Dockerfile.target.amazon-2 [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# First do the source builds
+@INCLUDE Dockerfile.target.sdist
+
+# This defines the dstribution base layer
```
Comment (Habbie) [inline builder-support/dockerfiles/Dockerfile.target.amazon-2:4]:
```suggestion
# This defines the distribution base layer
```

File: builder-support/dockerfiles/Dockerfile.target.centos-8 [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# First do the source builds
+@INCLUDE Dockerfile.target.sdist
+
+# This defines the dstribution base layer
```
Comment (Habbie) [inline builder-support/dockerfiles/Dockerfile.target.centos-8:4]:
```suggestion
# This defines the distribution base layer
```

File: builder-support/dockerfiles/Dockerfile.wforce [hunk 5]
```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:bionic as wforce
-
+FROM ubuntu:focal as wforce
```
Comment (Habbie) [inline builder-support/dockerfiles/Dockerfile.wforce:1]: Any reason you're using Ubuntu? For pdns/rec/dnsdist we specifically picked Debian.

File: builder-support/dockerfiles/Dockerfile.target.centos-8 (hunk 1 above)
Comment (neilcook) [reply builder-support/dockerfiles/Dockerfile.target.centos-8:16]: I just based it on the Centos-7 target Dockerfile, so the comments are from there. So no I didn't add the comments.

File: .travis.yml (hunk 2 above)
Comment (neilcook) [reply .travis.yml:81]: Yes and no. It should be in its own commit, which I will fix

File: builder-support/dockerfiles/Dockerfile.wforce (hunk 5 above)
Comment (neilcook) [reply builder-support/dockerfiles/Dockerfile.wforce:1]: I don't remember if there was a specific reason, although this is just for building packages, so does it matter?

Comment (Habbie) [reply builder-support/dockerfiles/Dockerfile.wforce:1]: Oh right, this is not the wforce Docker image. Ignore me :)

**Review by Habbie — APPROVED** [review]

**Pull request merged**